src/data/actions/search.js

```diff
@@ -0,0 +1,71 @@
+import 'whatwg-fetch';
```

Comment:
Not using fetch, we're using axios. We need to remove this import from other files at some point, but for now just remove this one.